docs/operator-manual/notifications/services/rocketchat.md

```diff
@@ -37,7 +37,7 @@ stringData:
   rocketchat-password: <password>
 ```
 
-8. Finally, use these credentials to configure the RocketChat integration in the `argocd-configmap` config map:
+8. Finally, use these credentials to configure the RocketChat integration in the `argocd-configmap` config map: 
```

Comment:
Weird. Not sure why `make codegen` would cause a regression.

Reply:
Looks like the `codegen` reverted a few those changes. It looks like these are generated from a [dependency](https://github.com/argoproj/argo-cd/blob/master/cmd/argocd/commands/admin/notifications.go) -- I will create a a pr for the notifications-engine